tests/functional/placementapi_controller_test.go

```diff
@@ -878,6 +878,134 @@ var _ = Describe("PlacementAPI controller", func() {
 		})
 	})
 
+	When("A PlacementAPI is created with topologyref", func() {
```

Comment:
Let use a different name than the test at L916. (Can be fixed in a follow up)

Reply:
Done